srcpkgs/glibc/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'glibc'
 pkgname=glibc
 version=2.36
-revision=1
+revision=2
```

Comment:
Just sanity-checking myself here: I'm doing a [custom rootfs build](https://github.com/SoMainline/void-bootstrap) courtesy of @JamiKettunen, and it's failing on `unresolved shlibs`. Should the revision suffix in `common/shlibs` have been bumped to `glibc-2.36_2`?

Reply:
no

that error probably means there's something being installed that was removed from the repositories at some point

Reply:
I did not want to spam a massive error message here for something that's likely broken on our end in either the script or our (dated!) package overlay(s).

In any case, substituting `glibc-2.36_1` for `glibc-2.36_2` in `common/shlibs` _fixes it_. I'll leave it to @JamiKettunen to sort out whether that's expected.